.openpublishing.redirection.json

```diff
@@ -1980,6 +1980,10 @@
             "source_path": "docs/standard/threading/autoresetevent.md",
             "redirect_url": "/dotnet/api/system.threading.autoresetevent"
         },
+        {
+            "source_path": "docs/standard/threading/eventwaithandle-autoresetevent-countdownevent-manualresetevent.md",
+            "redirect_url": "/dotnet/standard/threading/overview-of-synchronization-primitives"
```

Comment:
Another option is to redirect to the section of this article:
https://docs.microsoft.com/en-us/dotnet/standard/threading/overview-of-synchronization-primitives#thread-interaction-or-signaling

Reply:
Great suggestion. I think that redirecting to the specific section is a better alternative.,